google/google.go

```diff
@@ -69,6 +72,20 @@ func Setup(redirectURL, credFile string, scopes []string, secret []byte) {
 	}
 }
 
+// Setup the authorization path without a config file
+func SetupFromString(redirectURL, cLoginURL string, clientID string, clientSecret string, scopes []string, secret []byte) {
```

Comment:
Function should be `Setup`

Reply:
Are you suggesting @calston modify the existing `Setup`? Adding functionality with a new `SetupFromString` would only be a minor version update, opposed to a major version update. I'm in favor of their current implementation.

Reply:
Maybe `SetupWithOptions(o Options)`